<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcome">
     <widget class="QPushButton" name="create_wallet">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать кошелек</string>
      </property>
     </widget>
     <widget class="QPushButton" name="import_wallet">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Импортировать кошелек</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="importer">
     <widget class="QLineEdit" name="private_key_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>571</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставьте ваш секретный ключ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="public_key_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>571</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>571</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="line_open">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Открытый ключ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finish_import">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Импортировать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="creator">
     <widget class="QLineEdit" name="private_key">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>581</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="public_key">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>571</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Секретный ключ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Открытый ключ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generate">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Генерировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="show_private_key">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Показать секретный ключ</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="go">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Перейти к использованию</string>
      </property>
     </widget>
     <widget class="QLabel" name="warning_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="password_creator"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
